azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -173,7 +179,8 @@ def on_unsubscribed():
             self.transport.unsubscribe(topic=op.topic, callback=on_unsubscribed)
 
         else:
-            self.send_op_down(op)
+            # This code block should not be reached in correct program flow.
```

Comment:
If this is something that should not correctly be reached, we should add a logger debug here in case there's ever a bug where this branch is reached.

Reply:
An error will be raised

---
In reply to: [355024794](https://github.com/Azure/azure-iot-sdk-python/pull/409#discussion_r355024794) [](ancestors = 355024794)